src/services/organization-extension.js

```diff
@@ -270,6 +270,7 @@ module.exports = class orgExtensionsHelper {
 				instance: {
 					auto_save_interval: utils.convertToInteger(process.env.RESOURCE_AUTO_SAVE_TIMER),
 					note_length: utils.convertToInteger(process.env.MAX_RESOURCE_NOTE_LENGTH),
+					is_auth_token_bearer: process.env.IS_AUTH_TOKEN_BEARER === 'true',
```

Comment:
@priyanka-TL Do we need this since anyways default value will be set right?

Reply:
@aks30  based on this key frontend can decide they need to pass bearer or not